build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -333,12 +332,13 @@ multicluster:
 {{- with .Values.multicluster }}
 # Enable Antrea Multi-cluster Gateway to support cross-cluster traffic.
 # This feature is supported only with encap mode.
-  enable: {{ .enable }}
+  enableGateway: {{ .enableGateway }}
```

Comment:
The config in test-mc.sh need to be updated as well.
https://github.com/antrea-io/antrea/blob/main/ci/jenkins/test-mc.sh#L255

Reply:
Updated test-mc.sh. Thanks!